.github/workflows/main.yml

```diff
@@ -121,26 +94,24 @@ jobs:
 
   msrv:
     name: MSRV
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
     steps:
-      - uses: actions/checkout@master
-      - name: Install Rust
-        run: |
-          rustup update 1.60.0 --no-self-update
-          rustup default 1.60.0
-      - run: |
-          cargo test --verbose --manifest-path tests/Cargo.toml
-          cargo test --verbose --manifest-path tests/Cargo.toml --features kv
```

Comment:
Previously we check MSRV only for a subset.

Reply:
Previously we only applied it to a subset, but now that the key-value API is stable I’m happy to add one for it. What do you think @Thomasdezeeuw?

Reply:
Thanks for your information! I'd emphasize that ensuring MSRV for sval/serde would require bumping MSRV to 1.61, mainly for const fn. You may take it into consideration whether it's possible, since I see:

> This version is explicitly tested in CI and may be bumped in any release as needed. Maintaining compatibility with older compilers is a priority though, so the bar for bumping the minimum supported version is set very high. Any changes to the supported minimum version will be called out in the release notes.

Reply:
Technically a MSRV bump is a breaking change, but since 1.61 came out in May 2022, I think it's fine we increase it.

Reply:
I don't know why you say it is "technically" a breaking change. The `log` policy in the README even explicitly allows it, and that is the general practice of Rust libraries (although some do treat MSRV bumps as semver incompatible).

See also: https://doc.rust-lang.org/cargo/reference/rust-version.html#setting-and-updating-rust-version

Reply:
The topic has been discussed to death. Search the libs team repo or the api guidelines repo. I'm on mobile or I would link you. Increasing toolchain requirements is broadly not considered a breaking change.